ONSdigital/ras-party
# 269 ext user account unlock

# Motivation and Context
We needed to add an endpoint, which called notify gateway when a respondent first locks his/hers account. The endpoint tells notify to send an email with instructions about resetting and unlocking account.

# What has changed
New endpoint implemented

# How to test?
Run with fronstage PR [https://github.com/ONSdigital/ras-frontstage/pull/396#pullrequestreview-154559006](url)

Then lock your account, should see link appear in party logs, try resetting password and logging back in.
